rosdep/python.yaml

```diff
@@ -8608,6 +8608,16 @@ python3-retrying:
     pip:
       packages: [retrying]
   ubuntu: [python3-retrying]
+python3-rich-pip:
```

Comment:
could you add a python3-rich key instead?

- https://packages.ubuntu.com/search?keywords=python3-rich
- https://packages.debian.org/search?keywords=python3-rich

For older distros of ubuntu that don't have a binary package available, you can use pip

Reply:
See https://github.com/ros/rosdistro/pull/33017#pullrequestreview-1028893193

Reply:
Fair enough, thanks for taking a look at this PR! Going to abandon it and go a different route, perhaps custom rosdep rules installed on the local machine.